src/BuildingRegistry/Building/Events/BuildingUnit/BuildingUnitWasRetiredByParent.cs

```diff
@@ -7,14 +7,19 @@ namespace BuildingRegistry.Building.Events
     using ValueObjects;
 
     [EventName("BuildingUnitWasRetiredByParent")]
-    [EventDescription(
-        "Gebouweenheid werd gehistoreerd door een overkoepelende gebouweenheid (bvb. huisnummer > subadres).")]
+    [EventDescription("De gebouweenheid kreeg status 'gehistoreerd' door een overkoepelende gebouweenheid (bvb. huisnummer > subadres).")]
```

Comment:
Zou je hier wel subadres vermelden? Dat is een CRAB ding die niet meer bestaat in GRAR